### PR TITLE
mysql 8.0.26

### PR DESCRIPTION
**Pull request by bevanjkay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
Linux errors start at https://github.com/Homebrew/homebrew-core/pull/81580/checks?check_run_id=3120332272#step:7:2581 with
```
In file included from /usr/include/c++/5/bits/hashtable.h:35:0,
                 from /usr/include/c++/5/unordered_map:47,
                 from /tmp/mysql-20210721-8859-fb76ew/mysql-8.0.26/storage/perfschema/terminology_use_previous.cc:26:
/usr/include/c++/5/bits/hashtable_policy.h: In instantiation of ‘struct std::__detail::__is_noexcept_hash<PFS_class_type, std::hash<PFS_class_type> >’:
/usr/include/c++/5/type_traits:137:12:   required from ‘struct std::__and_<std::__is_fast_hash<std::hash<PFS_class_type> >, std::__detail::__is_noexcept_hash<PFS_class_type, std::hash<PFS_class_type> > >’
/usr/include/c++/5/type_traits:148:38:   required from ‘struct std::__not_<std::__and_<std::__is_fast_hash<std::hash<PFS_class_type> >, std::__detail::__is_noexcept_hash<PFS_class_type, std::hash<PFS_class_type> > > >’
/usr/include/c++/5/bits/unordered_map.h:100:66:   required from ‘class std::unordered_map<PFS_class_type, std::unordered_map<std::__cxx11::basic_string<char>, const char*> >’
/tmp/mysql-20210721-8859-fb76ew/mysql-8.0.26/storage/perfschema/terminology_use_previous.cc:183:20:   required from here
/usr/include/c++/5/bits/hashtable_policy.h:85:34: error: no match for call to ‘(const std::hash<PFS_class_type>) (const PFS_class_type&)’
  noexcept(declval<const _Hash&>()(declval<const _Key&>()))>
                                  ^
```

Try

```diff
diff --git a/Formula/mysql.rb b/Formula/mysql.rb
index f6a514d57a..dc7e16e7bf 100644
--- a/Formula/mysql.rb
+++ b/Formula/mysql.rb
@@ -35,11 +35,14 @@ class Mysql < Formula
 
   on_linux do
     depends_on "patchelf" => :build
+    depends_on "gcc"
   end
 
   conflicts_with "mariadb", "percona-server",
     because: "mysql, mariadb, and percona install the same binaries"
 
+  fails_with gcc: "5"
+
   def datadir
     var/"mysql"
   end
```

**Comment by carlocab** [comment]
Trying to build with GCC did not work. Help, @Homebrew/linux?

File: Formula/mysql.rb [hunk 1]
```diff
@@ -202,3 +205,17 @@ def plist
     system "#{bin}/mysqladmin", "--port=#{port}", "--user=root", "--password=", "shutdown"
   end
 end
+__END__
```
Comment (carlocab) [inline Formula/mysql.rb:208]:
```suggestion

__END__
```

**Comment by dawidd6** [comment]
```
==> FAILED
Missing libraries:
  unexpected (metadata_cache.so)
```

Hmm

**Review by dawidd6 — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thanks for your help @dawidd6

**Comment by carlocab** [comment]
🙏 @dawidd6

**Pull request closed without merging**